File: tasks/2024/jan/11.01/holiday.js
function seaSick(x) {
    const twentyPercent = x.length / 5;
    let count = 0;
    for (let i = 0; i < x.length - 1; i++) {
        if (x[i] !== x[i + 1]) {
            count += 1;
        }
    }
    if (count > twentyPercent) {
        return 'Throw Up';
    }
    return 'No Problem';
}


module.exports = seaSick;

File: tasks/20241/jan/11.01/holiday.py
def sea_sick(sea):
    waves_len = len(sea.split('_~')) - 1
    calm_len = len(sea.split('~_')) - 1
    sum_change = waves_len + calm_len
    print(sum_change)
    if sum_change / len(sea) > 0.2:
        return "Throw Up"
    return 'No Problem'